cpp/benchmarks/type_dispatcher/type_dispatcher_benchmark.cu

```diff
@@ -88,15 +88,12 @@ __global__ void host_dispatching_kernel(mutable_column_device_view source_column
 template <FunctorType functor_type>
 struct ColumnHandle {
   template <typename ColumnType>
-  void operator()(mutable_column_device_view source_column,
-                  int work_per_thread,
-                  rmm::cuda_stream_view stream = rmm::cuda_stream_default)
+  void operator()(mutable_column_device_view source_column, int work_per_thread)
   {
     cudf::detail::grid_1d grid_config{source_column.size(), block_size};
     int grid_size = grid_config.num_blocks;
     // Launch the kernel.
-    host_dispatching_kernel<functor_type, ColumnType>
-      <<<grid_size, block_size, 0, stream.value()>>>(source_column);
```

Comment:
Why would this have caused an error?

Reply:
The compile error is in the PR description. I believe an `include` would've also fixed this. 
Since the code is not actually using the stream for anything, I felt it more correct to just remove the defaulted stream parameter from this functor.